### ISSUE TITLE
Abandoned documentation removal

### ISSUE THREAD

**Issue by zregvart** [conforma/cli#1512]
With the documentation generation in #1500 and #1493 the `.adoc` files are generated in `docs/modules/ROOT/pages`. A document created prior to it's abandonment, for example if a command or a Rego builtin is removed, will not be removed.
The document generation code should keep track of any abandoned components and remove the corresponding Asciidoc files.

**Comment by simonbaird** [comment]
The lazy way would be to `rm -rf` all the generated files before regenerating them.

**Comment by zregvart** [comment]
> The lazy way would be to `rm -rf` all the generated files before regenerating them.

Yes, but we need a list of the ones we can remove, so fist step would be to figure out what `.adoc` files are generated vs which ones are handwritten. They're all in the same directory currently, so perhaps separating them somehow would be a good start.

The nav.adoc files get regenerated so we don't need to remove those.
